heron/tmaster/src/cpp/manager/tmaster.cpp

```diff
@@ -113,7 +108,7 @@ TMaster::TMaster(const std::string& _zk_hostport, const std::string& _topology_n
   tmaster_location_->set_master_port(master_port_);
   tmaster_location_->set_stats_port(stats_port_);
   DCHECK(tmaster_location_->IsInitialized());
-  EstablishTMaster(EventLoop::TIMEOUT_EVENT);
+  EstablishPackingPlan(EventLoop::TIMEOUT_EVENT);
```

Comment:
Why Packing Plan is fetched ahead of TMaster set? One invariant is that we should be confirmed as Tmaster before doing anything